pkg/resource/logging-config/alloy/test/logging-config.alloy.170_WC_default_namespaces_empty.yaml

```diff
@@ -328,3 +332,6 @@ alloy:
           path: /run/log/journal
       - name: alloy-tmp
         emptyDir: {}
+verticalPodAutoscaler:
```

Comment:
This looks related to https://github.com/giantswarm/logging-operator/pull/264
But how did the tests pass if the test golden files were wrong? Maybe the tests don't work and always pass?

Reply:
I think we're not running tests at all but I'm not sure

Reply:
I did run them myself so I know they pass but we probably want to add them to our pipeline

Reply:
Unit tests were never failing, fixed here: https://github.com/giantswarm/logging-operator/pull/267